proxy/src/test/java/org/apache/rocketmq/proxy/config/ConfigurationManagerTest.java

```diff
@@ -24,12 +24,6 @@
 
 public class ConfigurationManagerTest extends InitConfigTest {
 
-    @Test
-    public void testInitEnv() {
-        // configure proxy home by system env.
-        assertThat(ConfigurationManager.getProxyHome()).isEqualTo(mockProxyHome);
-    }
-
```

Comment:
Is there a mistake here? I can't pass this if I add this to the Bazel complie test.

Reply:
This has been fixed in #6329